components/clp-rust-utils/src/s3/client.rs

```diff
@@ -23,7 +30,8 @@ pub fn create_new_client(
         "clp-user",
```

Comment:
i feel this `provider_name` should be named as `clp-credentials-provider` or `clp-provider` to avoid confusion